net-firewall/ufw/ufw-0.36.ebuild

```diff
@@ -0,0 +1,194 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+DISTUTILS_IN_SOURCE_BUILD=1
+
+inherit bash-completion-r1 eutils linux-info distutils-r1 systemd
```

Comment:
It is my fault.Rmoved.

Reply:
I think the `LINGUAS` variable has been dropped from Gentoo since 2016. This only works if someone sets up a custom env.d file for it. 
Even so, installing localization files via USE flag is only acceptable when you need to download that localization file separately, since switching the IUSE will trigger a rebuild over a small file. You should just drop this huge hack and install all localization files unconditionally :P

Reply:
I can't call this one ebuild this is something like mixed c and bash scripting code:) But checked the code and really it do the job and works solid so i didn't want to change code block completely.

Interestingly dropping eutils doesn't break anything.If you have a time could you test it to being sure @juippis ?